src/router/router.js

```diff
@@ -30,6 +31,13 @@ export default new Router({
       path: '/events',
       name: 'events',
       component: Events,
+      props: true,
```

Comment:
Does this route have props?

Reply:
My bad, I think this was from something I was playing around with before.  I'll remove it!